GNUmakefile

```diff
@@ -52,12 +54,21 @@ testpercona:
 testtidb%:
 	$(MAKE) MYSQL_VERSION=$* MYSQL_PORT=34$(shell echo "$*" | tr -d '.') testtidb
 
-testtidb:
-	-docker run --rm --name test-tidb$(MYSQL_VERSION) -d -p $(MYSQL_PORT):4000 pingcap/tidb:v$(MYSQL_VERSION)
+bin/tiup:
```

Comment:
Could we do the same in Docker?

Your approach forces you to start considering arch, runs with `nohup`, depends on some mirror without chance to cache it, there may be potentially be issues with binary compatibility on newer GLIBC and so on.

Reply:
Yes, its possible with version 4.0 as described [here](https://docs.pingcap.com/tidb/v4.0/test-deployment-using-docker). But its not straight forward setup as tiup does. 
Starting from version 5.0 docker is not anymore supported method pulling up the cluster.  
This tool is needed only for acceptance testing. I'll glue it to am64 arch only. (similar like terraform binary fetch).

Reply:
Pingcap still publishes the docker-compose config: https://github.com/pingcap/tidb-docker-compose - but I admit it may be more complicated.

Still, if we don't want to use that - why should it run under `nohup`? Should it be left running in the background?

Reply:
Replace tiup with minimal docker setup. PTAL

Reply:
Thanks, this looks good. There is a minor issue there and I have / already had few more questions (replying you considered those is good enough for me). After that is done, I can merge it.